plugins/ui/src/deephaven/ui/_internal/RenderContext.py

```diff
@@ -41,6 +44,23 @@
 """
 
 
+@dataclass
+class ValueWithLiveness(Generic[T]):
+    """A value with an associated liveness scope, if any."""
+
+    value: T
+    liveness_scope: Union[LivenessScope, None]
+
+
+def value_or_call(value: T | None | Callable[[], T | None]) -> ValueWithLiveness[T]:
```

Comment:
Suggest making this an internal function, and adding a pydoc as wel.
```suggestion
def _value_or_call(value: T | None | Callable[[], T | None]) -> ValueWithLiveness[T]:
```